### ISSUE TITLE
Large data payloads in program deployment are being cut off

### ISSUE THREAD

**Issue by hathbanger** [versatus/lasr#88]
It looks as tho for a program that has a large data payload that lasr is cutting off a chunk of the registration transaction:

```
lasr   | 2024-04-01T09:14:14.665Z WARN  [lasr_actors::pending_transactions] added call transaction: 0x5d61712719e02899e92f0d9cc2e9a4f6b6c60e7fd993040fa424e2adbdc07125 to pre-call dependency graph
lasr   | 2024-04-01T09:14:14.665Z WARN  [lasr_compute::compute] Calling: runsc --rootless --network=none run -bundle ./containers/bafyreieu4cvk6fgi7nm3hi7sz57rqocp3tk7zdk53pune6bin42wwzfgei bafyreieu4cvk6fgi7nm3hi7sz57rqocp3tk7zdk53pune6bin42wwzfgei
lasr   | 2024-04-01T09:14:14.665Z WARN  [lasr_actors::executor] returning handle to executor
lasr   | 2024-04-01T09:14:14.665Z WARN  [lasr_actors::executor] result successful, placing handle in handles
lasr   | 2024-04-01T09:14:15.047Z WARN  [lasr_compute::compute] result from container: bafyreieu4cvk6fgi7nm3hi7sz57rqocp3tk7zdk53pune6bin42wwzfgei = "{\"computeInputs\":{\"version\":1,\"accountInfo\":{\"accountType\":{\"program\":\"0xc92ea48a2c05a806e73d881e8800613c14e690c7\"},\"programNamespace\":null,\"ownerAddress\":\"0x1234d4b33fa19bf655519f10e8fded9e31259b28\",\"programs\":{},\"nonce\":\"0x0000000000000000000000000000000000000000000000000000000000000000\",\"programAccountData\":{},\"programAccountMetadata\":{\"content_id\":\"bafyreieu4cvk6fgi7nm3hi7sz57rqocp3tk7zdk53pune6bin42wwzfgei\"},\"programAccountLinkedPrograms\":[]},\"transaction\":{\"transactionType\":{\"call\":\"0x0000000000000000000000000000000000000000000000000000000000000014\"},\"from\":\"0x1234d4b33fa19bf655519f10e8fded9e31259b28\",\"to\":\"0xc92ea48a2c05a806e73d881e8800613c14e690c7\",\"programId\":\"0xc92ea48a2c05a806e73d881e8800613c14e690c7\",\"op\":\"create\",\"transactionInputs\":\"{\\\"name\\\":\\\"Snakie snek\\\",\\\"symbol\\\":\\\"SNEK\\\",\\\"initializedSupply\\\":\\\"1\\\",\\\"totalSupply\\\":\\\"1\\\",\\\"to\\\":\\\"0xc92ea48a2c05a806e73d881e8800613c14e690c7\\\",\\\"imgUrl\\\":\\\"https://pbs.twimg.com/media/GJw6IvrXwAAYFqb?format=jpg&name=medium\\\",\\\"price\\\":\\\"1\\\",\\\"paymentProgramAddress\\\":\\\"0xc8e3d7cd8a9ec4a4f0738c045af56a10f7a5a973\\\"}\",\"value\":\"0x0000000000000000000000000000000000000000000000000000000000000000\",\"nonce\":\"0x0000000000000000000000000000000000000000000000000000000000000015\",\"v\":1,\"r\":\"0x36b2bb161aff1720fe3d048969181a6cbdd0e0a726db60d5e56cc18f8674fea6\",\"s\":\"0x48737f4e12197649f2e60468c3add2a4fb5951500bb04beed644bccf76769b56\"},\"op\":\"create\",\"contractInputs\":\"{\\\"name\\\":\\\"Snakie snek\\\",\\\"symbol\\\":\\\"SNEK\\\",\\\"initializedSupply\\\":\\\"1\\\",\\\"totalSupply\\\":\\\"1\\\",\\\"to\\\":\\\"0xc92ea48a2c05a806e73d881e8800613c14e690c7\\\",\\\"imgUrl\\\":\\\"https://pbs.twimg.com/media/GJw6IvrXwAAYFqb?format=jpg&name=medium\\\",\\\"price\\\":\\\"1\\\",\\\"paymentProgramAddress\\\":\\\"0xc8e3d7cd8a9ec4a4f0738c045af56a10f7a5a973\\\"}\"},\"instructions\":[{\"create\":{\"programNamespace\":\"this\",\"programId\":\"this\",\"programOwner\":\"0x1234d4b33fa19bf655519f10e8fded9e31259b28\",\"totalSupply\":\"0x0000000000000000000000000000000000000000000000000000000000000001\",\"initializedSupply\":\"0x0000000000000000000000000000000000000000000000000000000000000001\",\"distribution\":[{\"programId\":\"this\",\"to\":{\"address\":\"0xc92ea48a2c05a806e73d881e8800613c14e690c7\"},\"amount\":null,\"tokenIds\":[\"0x0000000000000000000000000000000000000000000000000de0b6b3a7640000\"],\"updateFields\":[{\"field\":\"data\",\"value\":{\"data\":{\"extend\":{\"type\":\"non-fungible\",\"imgUrl\":\"https://pbs.twimg.com/media/GJw6IvrXwAAYFqb?format=jpg&name=medium\",\"paymentProgramAddress\":\"0xc8e3d7cd8a9ec4a4f0738c045af56a10f7a5a973\",\"price\":\"1\",\"methods\":\"approve,create,burn,mint,update\",\"tokenMap\":\"{\\\"0x0000000000000000000000000000000000000000000000000de0b6b3a7640000\\\":{\\\"ownerAddress\\\":\\\"this\\\",\\\"data\\\":\\\"{\\\\\\\"imgUrl\\\\\\\":\\\\\\\"https://pbs.twimg.com/media/GJw6IvrXwAAYFqb?format=jpg&name=medium\\\\\\\"}\\\"}}\",\"snek\":\"PCFET0NUWVBFIGh0bWw+CjxodG1sIGxhbmc9ImVuIj4KPGhlYWQ+CiAgICA8bWV0YSBjaGFyc2V0PSJVVEYtOCI+CiAgICA8bWV0YSBuYW1lPSJ2aWV3cG9ydCIgY29udGVudD0id2lkdGg9ZGV2aWNlLXdpZHRoLCBpbml0aWFsLXNjYWxlPTEuMCI+CiAgICA8bWV0YSBodHRwLWVxdWl2PSJYLVVBLUNvbXBhdGlibGUiIGNvbnRlbnQ9ImllPWVkZ2UiPgogICAgPHNjcmlwdCBzcmM9Imh0dHBzOi8vY2RuLnRhaWx3aW5kY3NzLmNvbSI+PC9zY3JpcHQ+CiAgICA8dGl0bGU+U25ha2U8L3RpdGxlPgogICAgPHN0eWxlPgogICAgICAgIGJvZHkgewogICAgICAgICAgICBoZWlnaHQ6IDUwdmg7CiAgICAgICAgICAgIHdpZHRoOiAxMDB2dzsKICAgICAgICAgICAgZGlzcGxheTogZmxleDsKICAgICAgICAgICAganVzdGlmeS1jb250ZW50OiBjZW50ZXI7CiAgICAgICAgICAgIGFsaWduLWl0ZW1zOiBjZW50ZXI7CiAgICAgICAgICAgIG1hcmdpbjogMDsKICAgICAgICAgICAgYmFja2dyb3VuZC1jb2xvcjogYmxhY2s7CiAgICAgICAgfQoKICAgICAgICAjZ2FtZS1ib2FyZCB7CiAgICAgICAgICAgIGJhY2tncm91bmQtY29sb3I6ICNDQ0M7CiAgICAgICAgICAgIHdpZHRoOiA1MHZtaW47CiAgICAgICAgICAgIGhlaWdodDogNTB2bWluOwogICAgICAgICAgICBkaXNwbGF5OiBncmlkOwogICAgICAgICAgICBncmlkLXRlbXBsYXRlLXJvd3M6IHJlcGVhdCgyMSwgMWZyKTsKICAgICAgICAgICAgZ3JpZC10ZW1wbGF0ZS1jb2x1bW5zOiByZXBlYXQoMjEsIDFmcik7CiAgICAgICAgICAgIGJvcmRlci1yYWRpdXM6IDE2cHg7CiAgICAgICAgfQoKICAgICAgICAuc25ha2UgewogICAgICAgICAgICBiYWNrZ3JvdW5kLWNvbG9yOiAjZGIyNzc3OwogICAgICAgICAgICBib3JkZXI6IC4yNXZtaW4gc29saWQgYmxhY2s7CiAgICAgICAgfQoKICAgICAgICAuZm9vZCB7CiAgICAgICAgICAgIGJhY2tncm91bmQtY29sb3I6IGhzbCg1MCwgMTAwJSwgNTAlKTsKICAgICAgICAgICAgYm9yZGVyOiAuMjV2bWluIHNvbGlkIGJsYWNrOwogICAgICAgIH0KCiAgICAgICAgLm1vZGFsIHsKICAgICAgICAgICAgZGlzcGxheTogbm9uZTsgLyogSGlkZGVuIGJ5IGRlZmF1bHQgKi8KICAgICAgICAgICAgcG9zaXRpb246IGZpeGVkOyAvKiBTdGF5IGluIHBsYWNlICovCiAgICAgICAgICAgIHotaW5kZXg6IDE7IC8qIFNpdCBvbiB0b3AgKi8KICAgICAgICAgICAgbGVmdDogMDsKICAgICAgICAgICAgdG9wOiAwOwogICAgICAgICAgICB3aWR0aDogMTAwJTsgLyogRnVsbCB3aWR0aCAqLwogICAgICAgICAgICBoZWlnaHQ6IDEwMCU7IC8qIEZ1bGwgaGVpZ2h0ICovCiAgICAgICAgICAgIG92ZXJmbG93OiBhdXRvOyAvKiBFbmFibGUgc2Nyb2xsIGlmIG5lZWRlZCAqLwogICAgICAgICAgICBiYWNrZ3JvdW5kLWNvbG9yOiByZ2IoMCwwLDApOyAvKiBGYWxsYmFjayBjb2xvciAqLwogICAgICAgICAgICBiYWNrZ3JvdW5kLWNvbG9yOiByZ2JhKDAsMCwwLDAuNCk7IC8qIEJsYWNrIHcvIG9wYWNpdHkgKi8KICAgICAgICB9CgogICAgICAgIC5tb2RhbC1jb250ZW50IHsKICAgICAgICAgICAgYmFja2dyb3VuZC1jb2xvcjogI2ZlZmVmZTsKICAgICAgICAgICAgbWFyZ2luOiAxNSUgYXV0bzsgLyogMTUlIGZyb20gdGhlIHRvcCBhbmQgY2VudGVyZWQgKi8KICAgICAgICAgICAgcGFkZGluZzogMjBweDsKICAgICAgICAgICAgYm9yZGVyOiA0cHggc29saWQgIzg4ODsKICAgICAgICAgICAgd2lkdGg6IDgwJTsgLyogQ291bGQgYmUgbW9yZSBvciBsZXNzLCBkZXBlbmRpbmcgb24gc2NyZWVuIHNpemUgKi8KICAgICAgICAgICAgdGV4dC1hbGlnbjogY2VudGVyOwogICAgICAgICAgICBib3JkZXItcmFkaXVzOiAyNnB4OwogICAgICAgICAgICBmb250LXNpemU6IDI0cHg7CiAgICAgICAgICAgIGZvbnQtZmFtaWx5OiBzYW5zLXNlcmlmOwogICAgICAgICAgICBmb250LXdlaWdodDogYm9sZGVyOwogICAgICAgIH0KCiAgICAgICAgYnV0dG9uIHsKICAgICAgICAgICAgcGFkZGluZzogMTBweCAyMHB4OwogICAgICAgICAgICBjdXJzb3I6IHBvaW50ZXI7CiAgICAgICAgfQogICAgPC9zdHlsZT4KCiAgICA8c2NyaXB0PgogICAgICAgIGNvbnNvbGUubG9nKCJURVNUSU5HIikKICAgIDwvc2NyaXB0Pgo8L2hlYWQ+Cjxib2R5Pgo8ZGl2IGlkPSJ3YWxsZXQtY29ubmVjdC1tb2RhbCIgY2xhc3M9Im1vZGFsIHRleHQtYmxhY2siPgogICAgPGRpdiBjbGFzcz0ibW9kYWwtY29udGVudCI+CiAgICAgICAgPGltZyBzcmM9Imh0dHBzOi8vcGJzLnR3aW1nLmNvbS9tZWRpYS9HSnc2SXZyWHdBQVlGcWI/Zm9ybWF0PWpwZyZuYW1lPW1lZGl1bSIgY2xhc3M9InJvdW5kZWQtbWQgdy02NCBoLTY0IG0tYXV0byI+CiAgICAgICAgPHAgaWQ9Im1vZGFsLXRleHQiIGNsYXNzPSIiPkxBU1IgU25lazwvcD4KICAgICAgICA8cCBpZD0ibW9kYWwtYWRkcmVzcyIgY2xhc3M9InRleHQtc20gbWItNiI+PC9wPgogICAgICAgIDxidXR0b24gaWQ9Im1vZGFsLWJ1dHRvbiIgY2xhc3M9ImJnLXBpbmstNjAwIGJvcmRlci1ibGFjayBwLTYgcm91bmRlZC1tZCB0ZXh0LXdoaXRlIGhvdmVyOm9wYWNpdHktNTAiPkNvbm5lY3QgV2FsbGV0PC9idXR0b24+CiAgICA8L2Rpdj4KPC9kaXY+CjxkaXYgaWQ9ImdhbWUtYm9hcmQiPjwvZGl2Pgo8c2NyaXB0PgogICAgZG9jdW1lbnQuYWRkRXZlbnRMaXN0ZW5lcignRE9NQ29udGVudExvYWRlZCcsIGZ1bmN0aW9uICgpIHsKICAgICAgICB0cnkgewogICAgICAgICAgICBsZXQgbGFzdFJlbmRlclRpbWUgPSAwCiAgICAgICAgICAgIGxldCBnYW1lT3ZlciA9IGZhbHNlCiAgICAgICAgICAgIGNvbnN0IGdhbWVCb2FyZCA9IGRvY3VtZW50LmdldEVsZW1lbnRCeUlkKCdnYW1lLWJvYXJkJykKICAgICAgICAgICAgbGV0IGNvbm5lY3RlZCA9IGZhbHNlCiAgICAgICAgICAgIGxldCBjb25uZWN0ZWRBY2NvdW50ID0gbnVsbAogICAgICAgICAgICBjb25zdCBtb2RhbCA9IGRvY3VtZW50LmdldEVsZW1lbnRCeUlkKCd3YWxsZXQtY29ubmVjdC1tb2RhbCcpOwogICAgICAgICAgICBjb25zdCBtb2RhbEJ1dHRvbiA9IGRvY3VtZW50LmdldEVsZW1lbnRCeUlkKCdtb2RhbC1idXR0b24nKTsKICAgICAgICAgICAgY29uc3QgbW9kYWxUZXh0ID0gZG9jdW1lbnQuZ2V0RWxlbWVudEJ5SWQoJ21vZGFsLXRleHQnKTsKICAgICAgICAgICAgY29uc3QgbW9kYWxBZGRyZXNzID0gZG9jdW1lbnQuZ2V0RWxlbWVudEJ5SWQoJ21vZGFsLWFkZHJlc3MnKTsKICAgICAgICAgICAgbGV0IGdhbWVTdGFydGVkID0gZmFsc2U7CiAgICAgICAgICAgIG1vZGFsLnN0eWxlLmRpc3BsYXkgPSAnYmxvY2snOwogICAgICAgICAgICBtb2RhbEJ1dHRvbi5vbmNsaWNrID0gZnVuY3Rpb24oKSB7CiAgICAgICAgICAgICAgICBpZiAoIWdhbWVTdGFydGVkKSB7CiAgICAgICAgICAgICAgICAgICAgd2luZG93Py5sYXNyPy5yZXF1ZXN0QWNjb3VudCgpLnRoZW4oKGFjY291bnQpID0+IHsKICAgICAgICAgICAgICAgICAgICAgICAgY29uc29sZS5sb2coYWNjb3VudCk7CiAgICAgICAgICAgICAgICAgICAgICAgIGNvbm5lY3RlZEFjY291bnQgPSBhY2NvdW50CiAgICAgICAgICAgICAgICAgICAgICAgIGNvbm5lY3RlZCA9IHRydWU7IC8vIFRoaXMgc2hvdWxkIGJlIGRlY2xhcmVkIGF0IGEgaGlnaGVyIHNjb3BlIGlmIG5lZWRlZCBlbHNld2hlcmUKICAgICAgICAgICAgICAgICAgICAgICAgbW9kYWxUZXh0LnRleHRDb250ZW50ID0gYENvbm5lY3RlZCB0bzpgOwogICAgICAgICAgICAgICAgICAgICAgICBtb2RhbEFkZHJlc3MudGV4dENvbnRlbnQgPSBgJHthY2NvdW50LmFkZHJlc3N9YDsKICAgICAgICAgICAgICAgICAgICAgICAgbW9kYWxCdXR0b24udGV4dENvbnRlbnQgPSAnU3RhcnQgR2FtZSc7CiAgICAgICAgICAgICAgICAgICAgICAgIGdhbWVTdGFydGVkID0gdHJ1ZTsKICAgICAgICAgICAgICAgICAgICB9KS5jYXRjaCgoZXJyb3IpID0+IHsKICAgICAgICAgICAgICAgICAgICAgICAgY29uc29sZS5lcnJvcignQ29ubmVjdGlvbiBmYWlsZWQ6JywgZXJyb3IpOwogICAgICAgICAgICAgICAgICAgIH0pOwogICAgICAgICAgICAgICAgfSBlbHNlIHsKICAgICAgICAgICAgICAgICAgICBtb2RhbC5zdHlsZS5kaXNwbGF5ID0gJ25vbmUnOyAvLyBIaWRlIG1vZGFsIGFuZCBzdGFydCB0aGUgZ2FtZQogICAgICAgICAgICAgICAgICAgIHdpbmRvdy5yZXF1ZXN0QW5pbWF0aW9uRnJhbWUobWFpbik7CiAgICAgICAgICAgICAgICB9CiAgICAgICAgICAgIH07CgogICAgICAgICAgICBhc3luYyBmdW5jdGlvbiBtYWluKGN1cnJlbnRUaW1lKSB7CiAgICAgICAgICAgICAgICB0cnkgewoKICAgICAgICAgICAgICAgICAgICBpZiAoZ2FtZU92ZXIpIHsKICAgICAgICAgICAgICAgICAgICAgICAgaWYgKGNvbmZpcm0oJ1lvdSBsb3N0LiBQcmVzcyBvayB0byByZXN0YXJ0LicpKSB7CiAgICAgICAgICAgICAgICAgICAgICAgICAgICB3aW5kb3cubG9jYXRpb24gPSAnL2FwcHMvc25laycKICAgICAgICAgICAgICAgICAgICAgICAgfQogICAgICAgICAgICAgICAgICAgICAgICByZXR1cm4KICAgICAgICAgICAgICAgICAgICB9CiAgICAgICAgICAgICAgICAgICAgd2luZG93LnJlcXVlc3RBbmltYXRpb25GcmFtZShtYWluKQogICAgICAgICAgICAgICAgICAgIGNvbnN0IHNlY29uZHNTaW5jZUxhc3RSZW5kZXIgPSAoY3VycmVudFRpbWUgLSBsYXN0UmVuZGVyVGltZSkgLyAxMDAwCiAgICAgICAgICAgICAgICAgICAgaWYgKHNlY29uZHNTaW5jZUxhc3RSZW5kZXIgPCAxIC8gU05BS0VfU1BFRUQpIHJldHVybgogICAgICAgICAgICAgICAgICAgIGxhc3RSZW5kZXJUaW1lID0gY3VycmVudFRpbWUKICAgICAgICAgICAgICAgICAgICB1cGRhdGUoKQogICAgICAgIC2024-04-01T09:14:26.272Z WARN  [lasr_actors::pending_transactions] Attempting to clean pending graph
lasr   | 2024-04-01T09:14:41.273Z WARN  [lasr_actors::pending_transactions] Attempting to clean pending graph
lasr   | 2024-04-01T09:14:56.274Z WARN  [lasr_actors::pending_transactions] Attempting to clean pending graph
lasr   | 2024-04-01T09:15:11.275Z WARN  [lasr_actors::pending_transactions] Attempting to clean pending graph
```

**Comment by hathbanger** [comment]
update:

it would appear that any calls to programs aren't working, but sending is still good to go.

**Comment by hathbanger** [comment]
no longer an issue, believe it was fixed here: #101
